ios/FluentUI/Navigation/NavigationBar.swift

```diff
@@ -719,6 +725,19 @@ open class NavigationBar: UINavigationBar {
         }
     }
 
+    private func updateSubtitleView(for navigationItem: UINavigationItem?) {
+        guard let navigationItem = navigationItem, let subtitle = navigationItem.subtitle else {
+            // Use the default title view
```

Comment:
Should there be an enforcement here that subtitles are only added when there is also a title present?

Reply:
I'm not sure if we should do that. It definitely feels wrong to have a subtitle without a title, and we can definitely raise some sort of warning if that's the case, but I'm not sure if we should make this a requirement.

Reply:
It would be good to confirm with design about the intended behavior. If the intended design is to always have a title when a subtitle is present, we should enforce it. It's important to ensure our customers are guided to correct usage whenever possible.